charts/argo-cd/values.yaml

```diff
@@ -280,8 +280,10 @@ dex:
   ## Dex deployment container ports
   containerPortHttp: 5556
   servicePortHttp: 5556
+  servicePortHttpName: tcp-http
```

Comment:
Can we still use `http` for now as this change would be a breaking change?

Reply:
Sure! Default option could be still "http" - it will not work with istio, but at least we will have option to parameterize it.